backend/grades/resource/formatted/fall_2013.csv

```diff
@@ -1,2782 +1,2782 @@
-Abbreviation,CourseNum,CCN,SectionNum,CourseDesc,Instructor,Level,A+,A,A-,B+,B,B-,C+,C,C-,D+,D,D-,F,P,NP
```

Comment:
I don't think we need to modify these csv files for other terms. If we do proceed with these modified versions, we should be sure to understand what exactly changed.

Reply:
I am not sure that something did change - I think that all the files were overwritten so git thought I had basically deleted and re-added them - unless I am missing something while looking at the diffs.

Reply:
Or actually maybe they did change because of the updated .yaml file - do we not want to keep those changes?

Reply:
It does seem like the text is identical so it might be fine. The command for adding the data is specific to the term so unless we re-add data from past semesters, I don't see how this could cause any issues.

Reply:
Looks like his editor changed the file line endings from CRLF to LF or vise versa. I would just stick to whatever it was originally imo.